### PR TITLE
Ceph builders

### PR DESCRIPTION
**Pull request by ErwanAliasr1**
This branch is about creating a new role for having the role for the ceph infra + getting most of the requires updates to get edeploy runnng in 2016 ;)

### CODE REVIEW

File: build/sources/ceph-builders.upload [hunk 1]
```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+export OS_AUTH_URL="$OVH_AUTH_URL"
```
Comment (EmilienM) [inline build/sources/ceph-builders.upload]: I don't like much using OVH in the variables. I would rather re-use the same variable names.

**Comment by EmilienM** [comment]
Just a minor comment otherwise LGTM.
Nice work!

File: build/sources/ceph-builders.upload (hunk 1 above)
Comment (ErwanAliasr1) [inline build/sources/ceph-builders.upload]: That's a perfectly good remark. Even more that script should not be here. That's to specific for my usage. I'll move it out of edeploy.

**Comment by ErwanAliasr1** [comment]
@EmilienM I reworked the history and deleted the upload script which have nothing to be done here.

**Pull request merged**